rules.md

```diff
@@ -52,45 +52,40 @@ nav_order: 2
 
-**4.9** Use of hate speech, racial slurs, threats and any other language that could be considered harmful to a person or a group of people is not allowed.
+**4.7** Use of hate speech, racial slurs, threats and any other language that could be considered harmful to a person or a group of people is not allowed.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Discussion of such language, when it is clear that it is not being used for the intent of harm is okay, but should be undertaken with caution.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.7.1** Discussion of such language, when it is clear that it is not being used for the intent of harm is okay.
 
-**4.10** Explicit content of any form, including but not limited to: adult themes, erotic imagery, explicit innuendo, self harm and suicide, gore, shock content, etc. is not allowed.
+**4.8** Explicit content including: self harm and suicide, gore, shock content, etc. is not allowed.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.10.1** This includes in private messages to a player who did not consent to discussing them, or who no longer wishes to discuss them.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.8.1** This includes in private messages to a player who did not consent to discussing them, or who no longer wishes to discuss them.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.10.2** Discussing any of the above topics with players who are under the age of 18 is completely disallowed, even in private messages, even if they are contributing to the discussion and even outside of the server. **Any** disrespect to the age of consent will result in a non-negotiable ban.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.8.2** Discussing any of the above topics with players who are under the age of 18 is completely disallowed, even in private messages, even if they are contributing to the discussion and even outside of the server. **Any** disrespect to the age of consent will result in a non-negotiable ban.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.10.3** Whenever anything makes you uncomfortable, please contact a staff member privately.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.8.3** Whenever anything makes you uncomfortable, please contact a staff member privately.
 
-**4.11** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, redistributing paid software, scamming, ransomware, impersonation.
+**4.9** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, scamming, impersonation.
 
-**4.12** Automatically taking excessive amounts of items from public sources, such as ender storages, is considered theft.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Piracy and stealing through exploitation of bugs and other exploits in player software is allowed.
```

Comment:
Ransomware and attacks with the purpose of maliciously harming the player should be forbidden.

Reply:
This is not Windows, the only way to get ransomwares and viruses in CC is by giving physical access to other people (or public) to the computer or disk drives, exploiting ad-hoc programs that make use of networking (especially wireless networking), the latter of which even a proper antivirus cannot detect easily.
It is fine to me to allow attacks as long as player data is not affected maliciously.

Reply:
I updated it, do you think this is right?